docs/source/problems.rst

```diff
@@ -0,0 +1,93 @@
+========
+Problems
+========
+
+This document explains the core issues we see with the current state of Django
+and the ecosystem, which are the motivation for the project and influence the
+design of the resulting architecture.
+
+The modern web speaks multiple encodings and protocols
+======================================================
+
+When Django’s forms were designed in the mid 2000s, almost all communication
+with a website was encoded in HTML. Data was output by rendering to an HTML
+document using a templating system, and this was delivered to the client. Data
+was accepted using HTML ``<form>`` tag, and a Django ``Form``.
+
+In the modern web, HTML is still a major player but it shares the limelight
+with JSON, with communication generally taking place over some RESTful API. In
+the next 10 years of the web, who can say which other commincation encoding or
+communication architecture will become the new standard.
+
+Fundamentally, a data driven website such as the ones typically built using
+Django handles two major external processes. It delivers a representation of
+some data, and it receives requests to modify that data. In the current
+ecosystem, there are two distinct libraries most commonly used to achieve these
+core goals, and developers may mix and match between them within their
+projects. The first is core Django, which excels at rendering rich data into a
+document, and at accepting HTML form encoded data, validating it and storing
+it. The second is Django Rest Framework, which is optimised for building a
+developer friendly API, generally encoded in JSON. These two codebases both do
+a good job of their work, but the choice between them for a developer is made
+based on how you want to construct the architecture of your website. If you
+have built an HTML oriented website and then wish to add an API, you have to
+redefine many core parts of your logic in a similar bit subtly different
+system.
+
+Django needs to recognise that the web platform has moved on. It speaks
```

Comment:
@mjtamlyn on a serious note, what version of English are we choosing to write this in? Django docs are in American English?